packages/services/api/src/modules/organization/lib/personal-access-key.ts

```diff
@@ -0,0 +1,96 @@
+import * as Crypto from 'crypto';
+import bcrypt from 'bcryptjs';
+
+/**
+ * @module PersonalAccessKey
+ * Contains functions for generating an parsonal acces key.
+ */
+
+/**
+ * Payload within the access token.
+ */
+type DecodedAccessKey = {
+  /** UUID as stored within the database ("personal_access_tokens"."id") */
+  id: string;
+  /** string to compare against the hash within the database ("personal_access_tokens"."hash") */
+  privateKey: string;
+};
+
+/**
+ * Prefix for the personal access key.
+ * We use this prefix so we can quickly identify whether an organization access token.
+ *
+ * **hv** -> Hive
+ * **p**  -> Personal
+ * **1**  -> Version 1
+ */
+const keyPrefix = 'hvp1/';
```

Comment:
This file is identical to the `organization-access-key.ts` file, except for the key prefix which uses `p` instead of `o`.